include/haproxy/global-t.h

```diff
@@ -85,6 +85,7 @@
 #define GTUNE_LISTENER_MQ_OPT    (1<<28)
 #define GTUNE_LISTENER_MQ_ANY    (GTUNE_LISTENER_MQ_FAIR | GTUNE_LISTENER_MQ_OPT)
 #define GTUNE_QUIC_CC_HYSTART    (1<<29)
+#define GTUNE_MPTCP      		 (1<<30)
```

Comment:
Should we use `GTUNE_NO_MPTCP`, similar to `GTUNE_NO_QUIC`?

EDIT: but then if you do that, you should support a fallback in case of error when creating the socket: re-create a socket with the protocol set to 0